uaclient/config.py

```diff
@@ -678,6 +679,7 @@ def status(self, show_beta=False) -> "Dict[str, Any]":
         else:
             response = self._attached_status()
         response.update(self._get_config_status())
+        response = get_compat_schema(self, response)
```

Comment:
This is agreeing with Lucas' comment - Our migration is useful only for the following scenario.
* root user runs `ua status`
* uaclient is upgraded
* migration is run as root
* non-root user tries to run `ua status`

If we run it here, then it will either run as non-root and fail, or it is running when a root user ran `ua status` which isn't helpful

So I think we should plan on running this in `postinst`